toe/.urbitrc

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
This one is needed by `gulpfile.js`, it just needs to be pointed to where you pier is at

Reply:
You generally don't commit it though, since that info is very local to you, and not really useful on any other machine. We have it in the `/pkg/interface` [`.gitignore`](https://github.com/urbit/urbit/blob/4915ceb96bdd56f5b1f25504d1c68f8d41161540/pkg/interface/.gitignore#L69) for that reason, and might want to add it to this repo's `.gitignore` as well.

Reply:
ah cool, I'll update the README then so people know that one needs to be created before deploying.